ui/src/app/views/dataset/component.tsx

```diff
@@ -40,12 +43,34 @@ export const styles = {
   }
 };
 
+interface ValueSet {
+  domain: Domain;
+  values: DomainValue[];
+}
+
+interface DomainValuePair {
+  domain: Domain;
+  value: string;
+}
+
+export const ValueListItem: React.FunctionComponent <
+  {domainValue: DomainValue, onSelect: Function}> =
+  ({domainValue, onSelect}) => {
+    return <div style={{display: 'flex', color: 'black', height: '1.2rem'}}>
+      <input type='checkbox' value={domainValue.value} onClick={() => onSelect()}
```

Comment:
Without a `checked` prop, this input is uncontrolled and won't necessarily reflect the underlying state.

Reply:
Got it. Will add.